msteams-platform/get-started/first-message-extension.md

```diff
@@ -162,6 +162,23 @@ Deployment involves provisioning resources on an active Azure subscription and d
 
 </details>
 
+## Add a configuration page to your messaging extension on GitHub
```

Comment:
GitHub part in title is bit confusing. Could you please check if this can be changed?